### PR TITLE
wsl: add German translation

### PR DESCRIPTION
**Pull request by micschwarz**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/windows/wsl.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`wsl --distribution {{distribution}} {{shell_kommando}}`
+
+- Liste verfügbare Distributionen auf:
+
+`wsl --list`
+
+- Exportiere eine Distribution in eine `.tar` Datei:
+
+`wsl --export {{distribution}} {{pfad/für/tar_datei.tar}}`
```
Comment (pixelcmtd) [inline pages.de/windows/wsl.md:24]:
```suggestion
`wsl --export {{distribution}} {{pfad/zu/tar_datei.tar}}`
```

File: pages.de/windows/wsl.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Importiere eine Distribution von einer `.tar` Datei:
+
+`wsl --import {{distribution}} {{pfad/zum/installations_ordner}} {{pfad/zur/tar_datei.tar}}`
```
Comment (pixelcmtd) [inline pages.de/windows/wsl.md:28]:
```suggestion
`wsl --import {{distribution}} {{pfad/zu/installations_ordner}} {{pfad/zu/tar_datei.tar}}`
```

File: pages.de/windows/wsl.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`wsl --distribution {{distribution}} {{shell_kommando}}`
+
+- Liste verfügbare Distributionen auf:
+
+`wsl --list`
+
+- Exportiere eine Distribution in eine `.tar` Datei:
+
+`wsl --export {{distribution}} {{pfad/zu/tar_datei.tar}}`
```
Comment (marchersimon) [inline pages.de/windows/wsl.md:24]: I'm not sure what `distro_fs` means in the English page, but the `tar_` part here is reduntant anyways. How about
```suggestion
`wsl --export {{distribution}} {{pfad/zu/distro.tar}}`
```
Or just
```suggestion
`wsl --export {{distribution}} {{pfad/zu/datei.tar}}`
```

File: pages.de/windows/wsl.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Importiere eine Distribution von einer `.tar` Datei:
+
+`wsl --import {{distribution}} {{pfad/zu/installations_ordner}} {{pfad/zu/tar_datei.tar}}`
```
Comment (marchersimon) [inline pages.de/windows/wsl.md:28]:
```suggestion
`wsl --import {{distribution}} {{pfad/zu/installations_verzeichnis}} {{pfad/zu/tar_datei.tar}}`
```
Same for `tar_datei.tar` here.

File: pages.de/windows/wsl.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Ändere die WSL-Version einer bestimmten Distribution:
+
+`wsl --set-version {{distribution}} {{version}}`
+
+- Fahre das Windows Subsystem für Linux herunter:
```
Comment (marchersimon) [inline pages.de/windows/wsl.md:34]: Just a suggestion, but what do you think about
```suggestion
- Beeende das Windows Subsystem für Linux:
```

Comment (micschwarz) [reply pages.de/windows/wsl.md:34]: I think "Runterfahren" is more a OS thing and maybe better for WSL.

File: pages.de/windows/wsl.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`wsl --list`
+
+- Exportiere eine Distribution in eine `.tar` Datei:
+
+`wsl --export {{distribution}} {{pfad/zu/datei.tar}}`
+
+- Importiere eine Distribution von einer `.tar` Datei:
+
+`wsl --import {{distribution}} {{pfad/zu/installations_verzeichnis}} {{pfad/zu/tar_datei.tar}}`
```
Comment (marchersimon) [inline pages.de/windows/wsl.md:28]:
```suggestion
`wsl --import {{distribution}} {{pfad/zu/installations_verzeichnis}} {{pfad/zu/datei.tar}}`
```

**Review by pixelcmtd — APPROVED** [review]

File: pages.de/windows/wsl.md (hunk 6 above)
Comment (navarroaxel) [reply pages.de/windows/wsl.md:28]: I'll apply this change to merge.

Comment (micschwarz) [reply pages.de/windows/wsl.md]: Thanks!

**Pull request merged**